docs/cookbook/index.rst

```diff
@@ -149,13 +150,3 @@ Frontend
     frontend/webpack.rst
 
 .. include:: /cookbook/frontend/map.rst.inc
-
-Refund
-------
-
-.. toctree::
-    :hidden:
-
-    refund/refund-process.rst
```

Comment:
i am a bit concerned that payments will mix up with refunds in this case, but still customers can use `search` and find what they are looking for (very low priority comment imo) 🙂

Reply:
I wanted to be consistent with our current docs structure, where we have cookbook for customizing Credit Memo in Paymets section, IMO these two cookbooks should belong to the same section